src/store/mutations.js

```diff
@@ -108,7 +108,14 @@ export function setCurrentConditions(state, condition) {
     if (index === -1) {
         state.currentConditions.push(condition);
     } else {
-        Vue.set(state.currentConditions, index, condition);
+        let ConditionFixed = condition;
```

Comment:
It works as expected, but there are some things to improve:
1. use '===' instead of '==',
2. use lowercase first letter words for variables, capitalized for constructors.
3. try to avoid changing arguments, it can make it difficult to find errors in a code in a future
4. use constants instead of 'enum' and 'zygosity' strings, you can find it in the constants.js